resolvers/webpack/index.js

```diff
@@ -20,7 +20,15 @@ exports.interfaceVersion = 2
  * resolveImport('./foo', '/Users/ben/bar.js') => '/Users/ben/foo.js'
  * @param  {string} source - the module to resolve; i.e './some-module'
  * @param  {string} file - the importing file's full path; i.e. '/usr/local/bin/file.js'
- * TODO: take options as a third param, with webpack config file name
+ * @param  {object} settings - the webpack config file name, as well as cwd
+ * @example
+ * options: {
+ *  // Path to the webpack config
+ *  config: 'webpack.config.js',
+ *  // Path to be used to determine where to resolve webpack from
+ *  // (may differ from the cwd in some cases)
+ *  cwd: `${process.cwd()}`
```

Comment:
perhaps instead of `cwd`, the `config` path could default to `./webpack.config.js`, and if it's relative, resolved relative to `process.cwd()`; if absolute, used directly?

Reply:
I'm not sure the two are related. The `cwd` here is meant for the resolution of `webpack` the module here https://github.com/benmosher/eslint-plugin-import/pull/1503/files#diff-b3b04cb75e67b85c4a1489ef5ddf4110R174

It's unrelated to the config resolution. The idea being that your config and your webpack module may reside in two completely different areas on disk. Does that make sense? Maybe I'm misunderstanding the comment, so wanted to clarify.

Reply:
ah, i see what you mean
```suggestion
 *  cwd: process.cwd()
```